README.md

```diff
@@ -135,6 +135,22 @@ CMake's `find_package()` to be used.
 This library is using this mechanism if `nlohmann_json::nlohmann_json`-target
 does not exist.
 
+### Building with Bazel
+
+To build the library with bazel run:
+
+```bash
+bazel build src:json_schema_validator
+```
+
+The output files are placed in bazel-bin/src directory.
+
+To execute unit tests run:
+
+```bash
+bazel test test/...
+```
+
```

Comment:
It would be appreciated to have `.github/workflows` to do the build+test. If possible use any actions that wrap plain bazel. Could we leave it to you to find some relevant GH actions?